core/src/test/java/io/questdb/cairo/AbstractCairoTest.java

```diff
@@ -109,6 +110,9 @@ public class AbstractCairoTest {
             .withLookingForStuckThread(true)
             .build();
 
```

Comment:
There's a number of tests (e.g. IODispatcherTest) that don't participate in AbstractCairoTest hierarchy so one rule won't cover whole test suite .